src/components/axes/axis.ts

```diff
@@ -30,15 +30,14 @@ module Plottable {
     private _rescaleCallback: ScaleCallback<Scale<D, number>>;
 
     /**
-     * Constructs an axis. An axis is a wrapper around a scale for rendering.
+     * Constructs an Axis.
+     * An Axis is a visual representation of a Scale.
      *
      * @constructor
-     * @param {Scale} scale The scale for this axis to render.
-     * @param {string} orientation One of ["top", "left", "bottom", "right"];
-     * on which side the axis will appear. On most axes, this is either "left"
-     * or "bottom".
-     * @param {Formatter} Data is passed through this formatter before being
-     * displayed.
+     * @param {Scale} scale The Scale for this Axis to render.
+     * @param {string} orientation The orientation of the Axis ("top"/"bottom"/"left"/"right").
+     * on which side the Axis will appear.
+     * @param {Formatter} [formatter=Formatters.identity()] Tick values are passed through this Formatter before being displayed.
```

Comment:
Is `[name=defaultValue]` the style we are going with?  I'm ok with the choice here, I'm just verifying.

We might also want to look at how this looks in typedoc